src/plugins/iot/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@bimdata/iot-viewer-plugin",
+  "version": "0.0.1",
+  "description": "iot BIMData Viewer Plugin",
+  "scripts": {
+    "build": "rollup -c --environment BUILD:production",
+    "watch": "rollup --watch -c",
+    "prepublishOnly": "npm run build"
+  },
+  "main": "dist/iot.plugin.js",
+  "files": [
+    "dist"
+  ],
+  "keywords": [
+    "BIMData",
+    "Viewer",
+    "Plugin",
+    "iot"
+  ],
+  "dependencies": {
+    "chartist-plugin-tooltips-updated": "^0.1.3"
```

Comment:
Put all dependencies in devDependencies. They will be packaged anyway and programs depending on this plugin don't need to re-download the dependency